### ISSUE TITLE
[Terminal]: Extra info which only appear on Tooltip is not announced by Screen Reader for some controls throughout Terminal app.

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#12028]
### Windows Terminal version

1.12.3472.0

### Windows build number

10.0.22504.1010

### Other Software

**Test Environment:**
OS: Windows 11 Version Dev (OS Build 22504.1010)
App: Windows Terminal Preview
Screen Reader: Narrator
Tool: Accessibility Insights for Windows Version 1.1.1741.1

### Steps to reproduce

**Repro Steps:**
1. Open Windows Terminal.
2. Open Settings page using 'Ctr+,'.
3. Open Narrator using 'Win + Ctrl + Enter' keys.
4. Then navigate to all controls whose have tooltip like 'Open JSON file' and observe the issue.

**User Experience:**
Screen Reader users UX will not be good if they will miss some important info which only appear on tooltips.

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

**Attachments:**
[Extra info which only appear on Tooltip is not announced by Screen Reader for some controls throughout Terminal app..zip](https://github.com/microsoft/terminal/files/7773131/Extra.info.which.only.appear.on.Tooltip.is.not.announced.by.Screen.Reader.for.some.controls.throughout.Terminal.app.zip)

### Expected Behavior

Extra info which only appears on Tooltip should be announced by Screen Reader for all controls throughout Terminal app.
i.e. For 'Open JSON file', Tooltip is 'Open your Settings.json file, Alt+ Click to open defaults.json file' and Screen Reader should announce something like this 'Open your Settings.json file, Alt+ Click to open defaults.json file 1 of 1'

### Actual Behavior

Extra info which only appears on Tooltip is not announced by Screen Reader for some controls throughout Terminal app.
i.e. For 'Open JSON file', Tooltip is 'Open your Settings.json file, Alt+ Click to open defaults.json file' and Screen Reader is only announcing 'Open JSON file 12 of 12'

**Observation:** 
Issue repro with NVDA Screen Reader
Narrator users have option to read tooltips using shortcut key 'Caps lock + 0' keys.
